ewatercycle/forcing.py

```diff
@@ -0,0 +1,341 @@
+from esmvalcore.experimental import get_recipe
+from pathlib import Path
+
+FORCINGS = {
+    'ERA5': {
+        'dataset': 'ERA5',
+        'project': 'OBS6',
+        'tier': 3,
+        'type': 'reanaly',
+        'version': 1
+    },
+    'ERA-Interim': {
+        'dataset': 'ERA-Interim',
+        'project': 'OBS6',
+        'tier': 3,
+        'type': 'reanaly',
+        'version': 1
+    },
+}
+
+
+def update_hype(
+    data: dict,
+    *,
+    forcings: list = None,
+    startyear: int = None,
+    endyear: int = None,
+    shapefile: str = None,
+):
+    """
+    Update hype recipe data in-place.
+    
+    Parameters
+    ----------
+    forcings : list
+        List of forcings to use
+    startyear : int
+        Start year for the observation data
```

Comment:
```suggestion
        Start year for the forcing data
```
It's not necessarily observation data. We could also force a hydrological model with output data from a climate model, for example.